### ISSUE TITLE
ValueError while building path

### ISSUE THREAD

**Issue by mfschmidt** [nipreps/niworkflows#642]
I have processed a handful of subjects and receive the same error with each of them. I also saw the same error on neurostars (https://neurostars.org/t/query-regarding-dmriprep/18951?u=mike_schmidt), with no solution.

I have traced it back to where it appears a path is being requested from pybids with default patterns and valid entities, but it ends up raising a ValueError. (https://github.com/nipreps/niworkflows/blob/2600e4fe18012a852a9169bfac5804d3ad789eba/niworkflows/interfaces/bids.py#L583) All keys in the entities at the bottom of this report exist in https://github.com/nipreps/niworkflows/blob/master/niworkflows/data/nipreps.json, so I'm not sure why the exception.

The error at the bottom of the generated html report:

```
File: /out/dmriprep/sub-51081/log/20210524-020409_6c5f9005-dc32-494e-aed2-9f658ad05547/crash-20210524-035648-root-ds_report_eddy-b42e5acf-7866-4201-ae94-bdb214799ce1.txt
Working Directory: /work/dmriprep_wf/single_subject_51081_wf/dwi_preproc_ses_14458_wf/ds_report_eddy
Inputs:

    acquisition:
    atlas:
    base_directory: /out
    ceagent:
    check_hdr: True
    cohort:
    compress:
    data_dtype:
    datatype: figures
    density:
    desc: eddy
    direction:
    dismiss_entities:
    echo:
    extension:
    fmap:
    fmapid:
    from:
    hemi:
    in_file: ['/work/dmriprep_wf/single_subject_51081_wf/dwi_preproc_ses_14458_wf/eddy_report/report.svg']
    label:
    meta_dict:
    modality:
    mode:
    model:
    proc:
    reconstruction:
    recording:
    resolution:
    roi:
    run:
    scans:
    session:
    source_file: ['/work/dmriprep_wf/single_subject_51081_wf/dwi_preproc_ses_14458_wf/dwi_reference_wf/validate/sub-51081_ses-14458_dwi_valid.nii.gz']
    space:
    subject:
    subset:
    suffix:
    task:
    to:

Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 344, in _send_procs_to_workers
    self.procs[jobid].run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 516, in run
    result = self._run_interface(execute=True)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 635, in _run_interface
    return self._run_command(execute)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 741, in _run_command
    result = self._interface.run(cwd=outdir)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 434, in run
    runtime = self._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.7/site-packages/niworkflows/interfaces/bids.py", line 565, in _run_interface
    raise ValueError(f"Could not build path with entities {out_entities}.")
ValueError: Could not build path with entities {'subject': '51081', 'session': '14458', 'suffix': 'valid', 'extension': 'svg', 'desc': 'eddy', 'datatype': 'figures'}.
```

**Comment by effigies** [comment]
Looks like a bug in dmriprep setting the wrong `source_file`. Seems this was fixed in nipreps/dmriprep#169, so should be resolved in the next release.

**Comment by mfschmidt** [comment]
Thank you @effigies. I'll try pulling more recent code from their repo. Closing this issue.
